### DISCUSSION TITLE
New Look

### DISCUSSION THREAD

**Discussion by sir-scanda** [firefly-iii/firefly-iii#7520] [Firefly III]
Hi. Can You help me to open V3 Template in FFiii ... /v3 doesn't work... Thanks in advance.

**Comment by JC5** [comment]
What about it doesn't work?

**Comment by sir-scanda** [comment]
Hi

If i go to my server: http://192.168.50.100/firefly-iii/public/ works old template… http://192.168.50.100/firefly-iii/public/v3 gives blank page

**Reply by JC5** [reply]
Sorry, it doesn't really work with subdirectories like that.

**Comment by sir-scanda** [comment]
Ok.. is there any way to run new template?
